Content.Shared/Light/Components/SharedExpendableLightComponent.cs

```diff
@@ -23,12 +23,18 @@ public abstract partial class SharedExpendableLightComponent : Component
     [DataField("fadeOutDuration")]
     public float FadeOutDuration { get; set; } = 60 * 5f;
 
-    [DataField("spentDesc")]
-    public string SpentDesc { get; set; } = string.Empty;
-
     [DataField("spentName")]
     public string SpentName { get; set; } = string.Empty;
 
+    [DataField("refuelMaterialID")]
```

Comment:
```suggestion
    [DataField]
```
name gets automatically converted to camelCase, so this is redundant
same for the others

Reply:
Fixed in the next commit.